codegen/sdk-codegen/smithy-build.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Should this be committed? It'll be overwritten directly by the `generate-smithy-build` task.

Reply:
Oh, no. I thought I added this to .gitignore. Will fix.